style.css

```diff
@@ -440,3 +452,581 @@ body {
 
   /* Contact me ends */
 }
+
+@media only screen and (min-width: 601px) {
+  /* navbar starts */
+  .navbar {
+    display: flex;
+    justify-content: space-between;
+    background-color: #fff;
+    position: fixed;
+    top: 0%;
+    left: 0%;
+    right: 0%;
+    width: 100%;
+  }
+
+  .logo {
+    padding-left: 9.9%;
+    padding-top: 1%;
+    padding-bottom: 1%;
+  }
+
+  .union-bar {
+    display: none;
+  }
+
+  .logo a {
+    color: #6070ff;
+    text-decoration: none;
+    font-weight: 700;
+    font-size: 1.5rem;
+  }
+
+  .menu-options {
+    width: 35%;
+    list-style-type: none;
+    margin-block-start: 0;
+    margin-block-end: 0;
+    padding-top: 1.5%;
+    padding-right: 9.9%;
+    padding-inline-start: 0;
+    display: flex;
+    justify-content: space-around;
+  }
+
+  .menu-list {
+    color: #344563;
+    font-weight: 500;
+    font-size: 0.95rem;
+  }
+
+  /* navbar ends */
+
+  /* headline starts */
+  .headline {
+    margin-top: 4.4%;
+    padding-top: 17%;
+    padding-left: 6%;
+    padding-right: 6%;
+    padding-bottom: 10%;
+    background-image: url(./Properties/header-background-desktop.svg);
+    background-size: 100%;
+    background-repeat: no-repeat;
+    display: flex;
+    flex-direction: column;
+    background-color: #fff;
+    border-bottom-left-radius: 5rem;
+  }
```

Comment:
Kindly I suggest you stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the info of the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

>### Your Design
>
><img width="1433" alt="Screenshot 2023-05-04 at 11 46 15 AM" src="https://user-images.githubusercontent.com/91237525/236261606-3d1306cf-7f9d-4e95-a01b-af53c2998293.png">
>
>### The Figma design
>
><img width="692" alt="Screenshot 2023-05-04 at 11 58 36 AM" src="https://user-images.githubusercontent.com/91237525/236262402-00433e79-e289-4f6a-829a-a9bb5423b75e.png">